daily-challenges/2024-11/06/ththao27.spec.ts

```
@@ -0,0 +1,19 @@
+    if (!Array.isArray(numbers) || numbers.length === 0) {
+        throw new Error('Invalid input');
+    }
+    return numbers.map((num) => reversedNumber(num));
+}
+
+function reversedNumber(num: number): number {
+    if (typeof num !== 'number' || num <= 0 || !Number.isInteger(num)) {
+        throw new Error('Invalid input');
+    }
+    
+    const reverseNum = parseInt(num.toString().split('').reverse().join(''));
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)